Week01/Problem01/awillats_01.py

```diff
@@ -0,0 +1,23 @@
+
+# sum multiple of 3,5 less than 1000
+
+import math
+
+sum_var = 0
+
+num_max = 1000
+
+for i in range(num_max):
+    if math.fmod(i,3)==0:
```

Comment:
Instead of using the `math.fmod()` function, you could use the built-in modulo operator, `%`.  For example, this line could be re-written as
```
if (i % 3 == 0): 
```
Of course, it's kosher to use the `math.fmod` function, if you prefer.